source/native/symbulation.cc

```diff
@@ -79,33 +79,36 @@ int symbulation_main(int argc, char * argv[])
 
   //This parameter is redundant with HOST_REPRO_RES, SYM_HORIZ_TRANS_RES, and SYM_LYSIS_RES.
   //Configuring it adds another variable, but not another degree of freedom.
-  world.SetResPerUpdate(100);
+  world.SetResPerUpdate(100); // number of resources distributed per update
 
   int TIMING_REPEAT = config.DATA_INT();
   const bool STAGGER_STARTING_BURST_TIMERS = true;
 
   //Set up files
-  world.SetupPopulationFile().SetTimingRepeat(TIMING_REPEAT);
+  //world.SetupPopulationFile().SetTimingRepeat(TIMING_REPEAT);
 
+  if (config.LYSIS() == 1) {
+    world.SetupLysisFile(config.FILE_PATH()+"Lysis"+config.FILE_NAME()+".data").SetTimingRepeat(TIMING_REPEAT);
+  }
   world.SetupHostIntValFile(config.FILE_PATH()+"HostVals"+config.FILE_NAME()+".data").SetTimingRepeat(TIMING_REPEAT);
   world.SetupSymIntValFile(config.FILE_PATH()+"SymVals"+config.FILE_NAME()+".data").SetTimingRepeat(TIMING_REPEAT);
 
   
 
   //inject organisms
   for (size_t i = 0; i < POP_SIZE; i++){
-    Host *new_org;
+    Host *new_org; // Organisms are objects
```

Comment:
This isn't a necessary comment since that's standard in this kind of software, so let's remove it.